client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -25,14 +25,14 @@ class SimplePaymentsView extends Component {
 			return ( <QuerySimplePayments siteId={ siteId } productId={ productId } /> );
 		}
 
-		const { title, description, price, currency } = product;
+		const { title, description, price, currency, multiple } = product;
+
+		const paypalButtonImageUrl = '/calypso/images/simple-payments/preview-paypal-button-gray.png';
```

Comment:
In other parts in Calypso, image URLs like this are usually not a part of JS, but are referenced only in CSS, in a `background-image: url(...)` property.

The "G Suite" logo in the Email Settings is implemented in a smart way and could be an inspiration. Search for CSS class `email__add-google-apps-card-product-logo`. It has good accessibility, as it has a fallback text in case the image cannot be displayed or seen.

Is there any chance we could use a vector version of the image here? On Retina displays, it looks a bit blurry.

The layout breaks down on a narrow mobile viewport:
<img width="434" alt="mobile-viewport-paypal" src="https://user-images.githubusercontent.com/664258/28823118-57034daa-76bc-11e7-8520-11568f4e3ef7.png">
See how tiny the button becomes when there is not enough horizontal space?